webrtc/simplecall.html

```diff
@@ -108,9 +108,8 @@
 
   // This function starts the test.
   test.step(function() {
-    navigator.getUserMedia({ video: true, audio: true },
-        getUserMediaOkCallback,
-        failed('getUserMedia'));
+    navigator.mediaDevices.getUserMedia({ video: true, audio: true })
+      .then(getUserMediaOkCallback, failed('getUserMedia'));
```

Comment:
different PR please @jan-ivar. For now lets no longer show something the spec says exists only for backward compability reasons. If we keep showing the old stuff in examples it will take two decades to get rid of it.